src/pages/client/auth/Sign-up/styles.css

```diff
@@ -0,0 +1,98 @@
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+.auth-body{
```

Comment:
Bad copy-paste habbit to past things you don't need like auth-body which is proper to auth pages